Src/BootCamp.Chapter/Equipment.cs

```diff
@@ -7,58 +7,78 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Equipment
     {
+        private float _totalWeight;
+        private float _totalDefense;
+        private float _totalAttack;
+
+        //I haven't change those to properties cause it has so many if statements.
```

Comment:
It's not a very good argument though. The only time for method vs property when it should not be a property, is if it does too much (takes long too). This doesn't do too much.